### ISSUE TITLE
set up role-drinks meetup in hamburg

### ISSUE THREAD

**Issue by lassediercks** [lassediercks/role-drinks#1]
The ticket should be the first entry point in organising a role-drinks a11y meetup in Habmurg.

- - - - - - - 

### people who expressed interest 

 * Marco https://twitter.com/MarcoZehe

### people who offered help

 * Robin https://twitter.com/RobinDrexler "If you need a legal entity for invoices etc. or any other help with the event, let's talk." 

It can be closed when there is an executable plan.

**Comment by herschel666** [comment]
I'm interested, too. And I would help, when we hosted the meetup @ Jimdo.

**Comment by filtercake** [comment]
+1

**Comment by benpauer** [comment]
I'm in as well. Might be good if it's not just people from Jimdo though. Any ideas of how to get the word out and other people involved?

**Comment by filtercake** [comment]
@benpauer there is the web HH slack hub where most of hamburgs user groups organize: http://web-hh-slackin.herokuapp.com/

Be prepared to be asked "why yet another meetup" instead of integrating the topic into existing ones :)

**Comment by lassediercks** [comment]
We do have a location and a Date!!

It will be at the 17.8. in the xing office in hamburg

XING AG 
Dammtorstr. 30 
4. Stock 
20354 Hamburg 

They will also sponsor some drinks
https://twitter.com/Sabrittas organised that for us


next step is to set up the website

**Comment by filtercake** [comment]
re https://github.com/role-drinks/role-drinks.de/pull/3

while I am not a fan of meetup, I would in this case totally recommend using it. there is imo currently no other way of letting the right people in the right place know about it.

all active attendees of all the tech meetups will get notified (given the group is tagged accordingly).

Maybe one of our employers could pay the bill for a couple months?

**Comment by lassediercks** [comment]
as the pr is merged I consider this as done, 

I split the task of setiing up a meetup.com event into another issue: https://github.com/role-drinks/role-drinks.de/issues/4

here is my announcement tweet https://twitter.com/lassediercks/status/751036092183355393

I'm hyped!11!
